### PR TITLE
Fixes _child.Element null reference exception

### LINKED ISSUE #5700: ListView System.NullReferenceException
### Description

This only occurs on Android

I have a listview on a page in my app that is producing a System.NullReferenceException. This happens when its page is not at the top of the navigation stack. 

Stack trace: 

[MonoDroid] UNHANDLED EXCEPTION:
[MonoDroid] System.NullReferenceException: Object reference not set to an instance of an object.
[MonoDroid]   at Xamarin.Forms.Platform.Android.ListViewRenderer+Container.OnMeasure (System.Int32 widthMeasureSpec, System.Int32 heightMeasureSpec) [0x00032] in D:\a\1\s\Xamarin.Forms.Platform.Android\Renderers\ListViewRenderer.cs:513 
[MonoDroid]   at Android.Views.View.n_OnMeasure_II (System.IntPtr jnienv, System.IntPtr native__this, System.Int32 widthMeasureSpec, System.Int32 heightMeasureSpec) [0x00009] in <3faca578033c421aa590cc3addd3b6d8>:0 
[MonoDroid]   at (wrapper dynamic-method) System.Object.29(intptr,intptr,int,int)

### Steps to Reproduce

Narrowing this down to a repro; just wanted to get it up there.  

### Expected Behavior
I did not expect a null reference exception

### Actual Behavior
a null reference exception

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - Android: not sure how to answer this? I support multiple platforms and it seems to happen on all of them

### Reproduction Link

Working on it

### PR DESCRIPTION
**Pull request by Carbaholic**
### Issues Resolved ### 

- fixes #5700

### API Changes ###

 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [ ] Has automated tests 
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

**Comment by samhouts** [comment]
@Carbaholic Do you have a test or scenario in which this error was happening? Thanks!

File: Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs [hunk 1]
```diff
@@ -501,7 +501,8 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 
 			protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			{
-				if (_child == null)
+				if (_child == null ||
```
Comment (samhouts) [inline Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs]:
```suggestion
				if (_child?.Element == null)
```

File: Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs [hunk 2]
```diff
@@ -501,7 +501,8 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
...
+				    _child.Element == null)
```
Comment (samhouts) [inline Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs]:
```suggestion
```

Comment (Carbaholic) [reply Xamarin.Forms.Platform.Android/Renderers/ListViewRenderer.cs]: I will make that change as soon as I figure out how :)

I actually noticed a few things in that function that can be cleaned up.

**Comment by Carbaholic** [comment]
I do not have a test scenario. I'm running into the null reference exception in my current app and haven't been able to reproduce it. From the stack trace, however, I can clearly see that _child.Element is what is null.

**Comment by Carbaholic** [comment]
I am very new to contributing to open source projects. I've only tried once before with even less success than this try. Can someone give me some guidance? Why isn't anything happening with this pull request? Have I not done something I was supposed to do? 

Thanks!
Eric

**Comment by Carbaholic** [comment]
Is there documentation somewhere with a list of requirements before a pull request is accepted?

**Comment by samhouts** [comment]
@Carbaholic Hi! First, thank you for the pull request! 

We do review every pull request. We try to prioritize those which have a test case or a testing procedure that validates that the code change is necessary and resolves an outstanding issue. If you are able to provide more context in which your code change is needed, that would be helpful. 

Thanks for your patience

**Comment by Carbaholic** [comment]
This is a fix for Issue #5700.

I won't be able to create a test case until I've at least been able to create a minimum repro for the issue. This has proved difficult because the bug is dependent on the order through which you navigate through my app and creating a connection to a BLE device I have on site.

That said, it is a null reference exception that is clearly thrown at the line I mentioned.

**Comment by Carbaholic** [comment]
I am not authorized to view the details of the failed checks. What should I do next?

**Comment by samhouts** [comment]
@Carbaholic If you'd like to rebase your PR against 3.6.0 or 4.0.0, we can get the fix released earlier. Thanks!

**Comment by Carbaholic** [comment]
I'm having trouble rebasing because of the xamarin.forms.build submodule which I don't have access to. Should I need access to that in order to rebase?

**Comment by hartez** [comment]
@Carbaholic You shouldn't need access, no. What issues are you seeing when you try to rebase?

**Pull request merged**